app/assets/stylesheets/page.css.scss

```diff
@@ -0,0 +1,44 @@
+.image-modal-content {
+  max-width: 1200px;
+}
+
+.modal-image {
+  max-height: 100%;
+  max-width: 100%;
+}
+
+.show-pointer {
+  cursor: pointer;
+}
+
+.page-logo {
+  max-width: 100%;
+}
+
+.page-logo-container-index {
+  height: 50px;
+  width: 50px;
+  overflow: hidden;
+  float: left;
```

Comment:
do you "close" this float with "clear" later?

Reply:
Done